crates/exex/exex/src/notification.rs

```diff
@@ -22,6 +23,8 @@ pub enum ExExNotification {
         /// The old chain before reversion.
         old: Arc<Chain>,
     },
+    /// Finalized block header.
+    FinalizedBlock(Option<SealedHeader>),
```

Comment:
just to be consistent with other names

```suggestion
    BlockFinalized(Option<SealedHeader>),
```